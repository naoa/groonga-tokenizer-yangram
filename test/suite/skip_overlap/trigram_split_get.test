register tokenizers/yangram

tokenize TokenYaTrigramSplitSymbolAlpha "Tommorow is rainy" NormalizerAuto --mode GET
