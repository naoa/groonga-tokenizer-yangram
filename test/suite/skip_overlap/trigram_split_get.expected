register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register 3 --split_symbol 1 --split_alpha 1 --skip_overlap 1
[[0,0.0,0.0],"TokenYaTrigramSplitSymbolAlphaSkipOverlap"]
tokenize TokenYaTrigramSplitSymbolAlphaSkipOverlap "Tommorow is rainy" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "tom",
      "position": 0
    },
    {
      "value": "mor",
      "position": 3
    },
    {
      "value": "ow ",
      "position": 6
    },
    {
      "value": "is ",
      "position": 9
    },
    {
      "value": "rai",
      "position": 12
    },
    {
      "value": "iny",
      "position": 14
    }
  ]
]
